example.env

```diff
@@ -1,8 +1,12 @@
-# RPC provider eg. infura
-BLOCKCHAIN_PROVIDER_URL=https://ropsten.infura.io/v3/***
+# RPC provider eg. infura or BSC Testnet
+#BLOCKCHAIN_PROVIDER_URL=https://ropsten.infura.io/v3/***
+BLOCKCHAIN_PROVIDER_URL=https://data-seed-prebsc-2-s3.binance.org:8545/
```

Comment:
why would we comment out lines (#) in example.env?

Reply:
Removed.